core/src/apps/common/paths.py

```diff
@@ -338,6 +338,9 @@ def match(path: Bip32Path) -> bool:
 # https://github.com/trezor/trezor-firmware/issues/1749
 PATTERN_SEP5_LEDGER_LIVE_LEGACY = "m/44'/coin_type'/0'/account"
 
+PATTERN_CASA = "m/45'/[0-1000000]/[0-1000000]/[0-1]/[0-1000000]"
+PATTERN_CASA_EXPORT = "m/45'"
```

Comment:
I don't think we need to consider "casa export" at all. You can export the public keys via `GetPublicKey` that has no path restrictions (and we're bringing `EthereumGetPublicKey` in line with that in #2410) and the export path should not be valid in any other context.

Reply:
Perfect! I'll try that out